### PR TITLE
fix(changes): add 1.13.7 release notes

### PR DESCRIPTION
**Pull request by Poltergeist**
This adds 1.13.7 release notes and adds all the changes.

## High-level description

We missed to create change log title for 1.13.7 I fixed this.

## Corresponding DC/OS tickets (required)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by Poltergeist** [comment]
@mesosphere-mergebot label Ready For Review

File: CHANGES.md [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+### Fixed and improved
+
+* DC/OS overlay networks should be compared by-value. (DCOS_OSS-5620)
+
+* Drop labels from Lashup's kv_message_queue_overflows_total metric. (DCOS_OSS-5634)
+
+* Reserve all agent VTEP IPs upon recovering from replicated log. (DCOS_OSS-5626)
+
+* Use Golang 1.10.8 to build CockroachDB. (DCOS-61502)
+* [Mesos] Support quoted realms in WWW-Authenticate header (DCOS-61529)
```
Comment (asekretenko) [inline CHANGES.md]: IIUC, this is covered by the
`Updated to Mesos [1.8.2-dev](https://github.com/apache/mesos/blob/c7c716dbc9ee4363ba6267591585b9984d8920b8/CHANGELOG)`

above, isn't it?

Comment (Poltergeist) [reply CHANGES.md]: I don't know I just took what was in the file before andd added it to 1.13.7.

**Comment by DominikDary** [comment]
@Poltergeist is this still needed or can this be closed?

**Comment by vinodkone** [comment]
I've close this since it's stale.

**Pull request closed without merging**